src/main/java/io/turntabl/employementprofilingsystem/Controllers/AuthController.java

```diff
@@ -42,6 +52,7 @@
                         new Object[]{email},
                         BeanPropertyRowMapper.newInstance(Employee.class)
                 );
+                rootSpan.setTag("db.statement", "select * from employee where employee_email = " + email);
```

Comment:
so I think this and the template query above should be one string, so that if someone comes along and changes the query on line 51 it changes in the trace too.